app/views/mailers/digest_mailer/digest_email.html.erb

```diff
@@ -45,7 +45,7 @@
       <% elsif tip_rand == 3 %>
         You're a better <%= SiteConfig.community_member_label %> today than you were yesterday.
       <% elsif tip_rand == 4 %>
-        <% if @user.articles.published.any? %>
+        <% if @user.articles_count > 0 %>
```

Comment:
Does this only include published articles? Because I don't see anything special in the `Article` model to suggest the drafts are not included in this count. Is that okay?

Reply:
This includes drafts. I figured based on the wording and that this is only chosen 1/5 of the time including drafts in this count is not a huge deal. It's also the very bottom of the email so I don't think it is consumed very much and the difference between first and next is not much in my mind.

Reply:
Fine by me! Let's see if Vaidehi comes up with a compelling reason not to use it. Otherwise we'll have to add a new column for the published articles and point counter culture to it 👎

Reply:
i think it's probably okay to do this; i would vote that, if anything, we change the wording rather than add a new column for this. but imho, i think the wording here works 👍